soccer/gameplay/evaluation/touchpass_positioning.py

```diff
@@ -0,0 +1,100 @@
+import robocup
+import constants
+import main
+import evaluation.passing
+
+## The Touchpass Positioner finds the best location within a rectangle
+#
+# Example usage:
+# tpos = TouchpassPositioner()
+# best_pt = tpos.evaluate(ball.pos, robot2.pos)
+class TouchpassPositioner:
```

Comment:
What are your thoughts on changing this from a class to just a set of functions?  It seems like there isn't really any state to keep around, just some evaluation calls.